casper/src/main/scala/coop/rchain/casper/util/ProtoUtil.scala

```diff
@@ -246,6 +246,8 @@ object ProtoUtil {
     ByteString.copyFrom(Base16.decode(string))
 
   def basicDeployString(id: Int): DeployString = {
+    //todo this shoule be removed once we assign the deploy with exact user
```

Comment:
Can you make this

// TODO:

Later on we want to grep for this pattern exactly when we do clean up

Reply:
Done